### ISSUE TITLE
Run check links only periodically

### ISSUE THREAD

**Issue by dalonsoa** [SGIModel/MUSE_OS#147]
Currently, links in the documentation are checked with every push to a PR. This does not makes sense as the checks are done on the published version of the documentation, which only changes when there's a change in the main branch (and only after it has had time to re-deploy the documentation).

The action should be change so it runs only periodically (eg. once a week) rather than with every push/PR. An alternative method, if needed, could be implemented to check that new versions of the documentation also have working links, but that would be a separate issue.
